pkg/types/validation/installconfig.go

```diff
@@ -699,3 +703,36 @@ func validateFIPSconfig(c *types.InstallConfig) field.ErrorList {
 	}
 	return allErrs
 }
+
+// validateCapabilities checks if additional, optional OpenShift components are specified in the
+// install-config to be included in the installation.
+func validateCapabilities(c *configv1.ClusterVersionCapabilitiesSpec, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	allAvailableCapabilities := map[string]bool{}
```

Comment:
Use a [set](https://github.com/kubernetes/apimachinery/blob/master/pkg/util/sets/string.go) instead of a map. This will obviate the need for the `allCaps` variable, too.
```suggestion
	allAvailableCapabilities := sets.NewString()
```

Reply:
+1.